composer.json

```diff
@@ -22,9 +22,10 @@
         "prooph"
     ],
     "require": {
-        "prooph/event-store": "^7.0",
-        "prooph/pdo-event-store": "^1.0",
-        "prooph/snapshot-store": "^1.0"
+        "prooph/event-store": "^7.2",
+        "prooph/pdo-event-store": "^1.4.1",
+        "prooph/snapshot-store": "^1.2",
+        "phunkie/phunkie": "^0.1"
```

Comment:
You should lock the dependency to a more specific version (e.g. 0.10) since it's below beta, not to be surprised with non-BC changes.